python/pyproject.toml

```diff
@@ -68,6 +68,9 @@ scikit-learn = [
 fugue = {version = "^0.8.1", optional = true}
 requests = "^2.27"
 
+# NLP related dependencies
+pybloomfiltermmap3 = {version = "^0.5.7", python = ">=3.5", optional = true}
```

Comment:
While this is an optional nlp specific dependency, I have concerns about pulling this package in, are there alternatives? This would require more testing on different platforms / os / ISA before we could pull this in as I'm not sure this works on the platforms, os, and ISA matrix that whylogs supports.